server/src/main/java/org/apache/iotdb/db/auth/entity/PrivilegeType.java

```diff
@@ -46,6 +45,7 @@
   DROP_TRIGGER,
   START_TRIGGER,
   STOP_TRIGGER,
+  TTL,
   ALL;
```

Comment:
All new privileges should be appended to the tail as they are serialized according to their ordinals, changing the order may introduce compatibility problems.

Reply:
By the way, have you mentioned the new privileges in the document?

Reply:
I mentioned this new privilege in docs/UserGuide/Administration-Management/Administration.md.

Reply:
> All new privileges should be appended to the tail as they are serialized according to their ordinals, changing the order may introduce compatibility problems.

So why you approve the PR though it is not changed?